packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-vdsm/tasks/libvirt_presets_configure.yml

```diff
@@ -14,5 +14,5 @@
         disable virtstoraged.socket
         disable virtproxyd.socket
         enable libvirtd.service
-
-  when: el_ver|int >= 9
+      mode: preserve
```

Comment:
When the dest file doesn't exist, system umask is used.  This calls for problems, is there a reason not to specify the file mode directly here?

Reply:
fixed